core/src/test/scala/sttp/openai/requests/audio/AudioCreationDataSpec.scala

```diff
@@ -0,0 +1,28 @@
+package sttp.openai.requests.audio
+
+import org.scalatest.EitherValues
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+import sttp.openai.fixtures
+import sttp.openai.json.SttpUpickleApiExtension
+
+class AudioCreationDataSpec extends AnyFlatSpec with Matchers with EitherValues {
+  "Given audio generation response as Json" should "be properly deserialized to case class" in {
+    import sttp.openai.requests.audio.AudioResponseData.AudioResponse
+    import sttp.openai.requests.audio.AudioResponseData.AudioResponse._
+
+    // given
+    val jsonResponse = fixtures.AudioFixture.jsonResponse
+
+    val expectedResponse = AudioResponse(
+      "Imagine the wildest idea that you've ever had, and you're curious about how it might scale to something that's a 100, a 1,000 times bigger. This is a place where you can get to do that."
+    )
+
+    // when
+    val givenResponse = SttpUpickleApiExtension.deserializeJsonSnake.apply(jsonResponse)
```

Comment:
**nitpick:** Let's call it `deserializedJsonResponse`, to avoid using a confusing occurence of the word `given` in a `when` section.